medusa/providers/torrent/rss/nyaa.py

```diff
@@ -143,5 +143,21 @@ def parse(self, data, mode):
 
         return items
 
+    def _get_season_search_strings(self, episode):
+        search_string = {
+            'Season': []
+        }
+        season_search_keywords = ('batch', 'complete', 'season')
```

Comment:
Then I suggest we add an attribute to the providers named `season_pack_keywords` (or something else). And if that attribute exists, we use the season_search_string creation, as shown in this PR?